native/metro.config.js

```diff
@@ -1,52 +1,64 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 /**
  * Metro configuration for React Native
  * https://github.com/facebook/react-native
  *
  * @format
  */
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 const path = require('path')
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 const Resolver = require('metro-resolver')
 
-module.exports = {
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config')
+// eslint-disable-next-line import/no-extraneous-dependencies
+const defaultSourceExts = require('metro-config/src/defaults/defaults').sourceExts
+// eslint-disable-next-line import/no-extraneous-dependencies
+const defaultAssetExts = require('metro-config/src/defaults/defaults').assetExts
+
+module.exports = mergeConfig(
+  getDefaultConfig(__dirname),
   // https://github.com/facebook/react-native/issues/21310#issuecomment-544071895
   // https://github.com/facebook/metro/issues/1#issuecomment-453450709
   // Per default metro is looking for the babel dependencies in the node_modules of the corresponding package.
   // This leads to problems in the build-configs module as the babel dependencies are only in the native .
-  resolver: {
-    extraNodeModules: new Proxy(
-      {},
-      {
-        get: (target, name) => {
-          if (name === 'build-config-name') {
-            const buildConfigName = process.env.BUILD_CONFIG_NAME || 'integreat-test-cms'
-            // Proxy the (non-existing) module 'build-config-name' to the name of the right build config.
-            // Passing environment variables is not possible without either babel or post processing otherwise.
-            return path.resolve(__dirname, '../build-configs', buildConfigName, 'build-config-name/index.ts')
-          }
+  {
+    resolver: {
+      assetExts: defaultAssetExts.filter(ext => ext !== 'svg'),
+      sourceExts: [...defaultSourceExts, 'svg'],
+      extraNodeModules: new Proxy(
+        {},
+        {
+          get: (target, name) => {
+            if (name === 'build-config-name') {
+              const buildConfigName = process.env.BUILD_CONFIG_NAME || 'integreat-test-cms'
```

Comment:
why only 'integreat-test-cms' and not also malte?

Reply:
This is a fallback if the env variable is not set I think and the test cms is just the default here I think

Reply:
Think so, too. This was just copy/pasted :)